wls-gui-wahllokalsystem/src/composables/wahlvorbereitung/wahlvorbereitungService.ts

```diff
@@ -142,11 +150,55 @@ export function useWahlvorbereitungService() {
     }
   }
 
+  async function getBriefwahlvorbereitung(
+    wahlbezirkID: string
+  ): Promise<Wahlvorbereitung> {
+    try {
+      return await briefwahlvorbereitungControllerAPI
+        .getBriefwahlvorbereitung(wahlbezirkID)
+        .then((response) => toBriefwahlvorbereitungModel(response.data));
+    } catch (error) {
+      userNotificationService.addNotification(
+        "Fehler beim Laden der Wahlvorbereitung.",
```

Comment:
Wäre ananlog zur UWB-Meldung

```suggestion
        "Fehler beim Laden der Briefwahlvorbereitung.",
```